packages/stockfish-git/stockfish-git.pacscript

```diff
@@ -0,0 +1,21 @@
+name="stockfish-git"
+gives="stockfish"
+pkgdesc="The Stockfish chess engine, built from the git repository"
```

Comment:
don't include the part about it being from git.

Reply:
okie dokie

Reply:
Is this good